src/main/java/sirius/pasta/tagliatelle/macros/IfNotDefinedMacro.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.pasta.tagliatelle.macros;
+
+import sirius.kernel.commons.Strings;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.tokenizer.Position;
+import sirius.pasta.noodle.Environment;
+import sirius.pasta.noodle.compiler.CompilationContext;
+import sirius.pasta.noodle.compiler.ir.Node;
+import sirius.pasta.noodle.macros.BasicMacro;
+import sirius.pasta.tagliatelle.rendering.GlobalRenderContext;
+import sirius.pasta.tagliatelle.rendering.LocalRenderContext;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+/**
+ * Checks whether the given guard is defined or not.
```

Comment:
**Optional:** Link guard to the new toggle/has guard method in the context?
Or add them as `@see` marks